### PR TITLE
adding in a blog on rse showcase

### PR DESCRIPTION
**Pull request by Sparrow0hawk**
adding in a blog post on the ramp ua project

### CODE REVIEW

**Comment by Sparrow0hawk** [comment]
@NGRhodes @jhodrien appreciate a quick check of how this looks before approving to merge

File: blog/_posts/2021-03-05-RSEshowcaseRAMPUA.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+#### 4. Holding weekly developer stand ups
+
+  Working with a team distributed across the UK and all working on different aspects of the code meant it was crucial to instigate good practices around distributed team working. At the core of this was organising a weekly stand-up Microsoft Teams Call where everyone working on the code base would join for just half an hour, once a week to go through what they'd been up to, what their plans were and to identify anything blocking or potentially blocking their progress. This helped everyone keep pace with the project and make sure others were able to know what was happening and whether there was anything they could contribute. To aid with this we initially used the [Teams feature](https://github.com/team) within GitHub organisations but more on that in [6.](./#6-making-the-most-out-of-github-repository-tools)
```
Comment (NGRhodes) [inline blog/_posts/2021-03-05-RSEshowcaseRAMPUA.md:47]: The link at the end of the paragraph does not work - appears to be no bookmark.

Comment (Sparrow0hawk) [reply blog/_posts/2021-03-05-RSEshowcaseRAMPUA.md]: Good spot, have resolved this in 17181d5

**Comment by NGRhodes** [comment]
Tested locally, looks fine.

**Pull request merged**